sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/aio/client.py

```diff
@@ -147,15 +147,15 @@ class SecretClient(AsyncKeyVaultClientBase):
             error_map={404: ResourceNotFoundError},
             **kwargs
         )
-        return SecretAttributes._from_secret_bundle(bundle)  # pylint: disable=protected-access
+        return SecretProperties._from_secret_bundle(bundle)  # pylint: disable=protected-access
 
     @distributed_trace
-    def list_secrets(self, **kwargs: "**Any") -> AsyncIterable[SecretAttributes]:
+    def list_secret_properties(self, **kwargs: "**Any") -> AsyncIterable[SecretProperties]:
```

Comment:
Similarly, this should remain `list_secrets`

Reply:
switched back to list_secrets, thanks!